src/Resources/public/js/easyadmin-extension.js

```diff
@@ -1,87 +1,95 @@
-function reloadEmbeddedList(identifier, toggleBaseUrl)
-{
-  var containerPrefix = '.embedded-list[for="'+identifier+'"]';
+function reloadEmbeddedList(identifier, toggleBaseUrl) {
+    var containerPrefix = '.embedded-list[for="'+identifier+'"]';
```

Comment:
Please follow project indentation size for JS = 2 spaces. see https://github.com/alterphp/EasyAdminExtensionBundle/blob/master/.editorconfig

Reply:
Most code editors can consider `.editorconfig` files

Reply:
And changing indentation size in a commit that have real modifications makes it particularly hard to read for reviewers ;-)